imports/server/setup.ts

```diff
@@ -227,6 +227,10 @@ Meteor.publish('hasUsers', function () {
   this.ready();
 });
 
+const cacheDuration = 5000; // Cache guild listings for 5 seconds to avoid hitting ratelimits
+let cachedDiscordGuildsTimestamp = 0;
+let cachedDiscordGuilds: any[] = [];
```

Comment:
I think I'm close to requesting a helper for this

Reply:
yeah I can probably extract one before I add anything more